### DISCUSSION TITLE
lolMiner 1.40

### DISCUSSION THREAD

**Discussion by papnikol** [Lolliedieb/lolMiner-releases#925] [General]
I am getting a virus threat with 1.40
I know this is typical for miners, but I was not getting it on 1.39. I am wondering what changed...

**Comment by riuamasa** [comment]
same here

**Comment by kebecode** [comment]
The miner is getting an error every 18-22 minutes in HiveOs with https pools and gets restarted

![image](https://user-images.githubusercontent.com/1355330/148446030-d1f2bdcc-22f9-4bc2-8b58-77ebce555f80.png)

in Windows that error doesn't happen
